css/offline.less

```diff
@@ -205,10 +205,12 @@
     }
 
     .caching-spinner .wait-spinner {
-        width: 70px;
-        height: 70px;
+        position: absolute;
+        width: 30px;
+        height: 30px;
         margin: 0;
-        left: 0
+        left: 20px;
+        top: 22px;
```

Comment:
Does this work correctly on all screen sizes?

Reply:
Yep, just stays in the corner